pandas/plotting/_style.py

```diff
@@ -114,8 +114,11 @@ def _get_standard_colors(num_colors=None, colormap=None, color_type='default',
             import random
 
             def random_color(column):
+                rstate = random.getstate()  # to avoid resetting the seed
```

Comment:
Point to the issue number.

Reply:
Also, since you're changing the behavior, let's document it.  A brief doc-string would be good here.